src/DependencyInjection/BaldinofRoadRunnerExtension.php

```diff
@@ -239,10 +269,178 @@ private function configureKv(array $config, ContainerBuilder $container): void
+            /** @phpstan-ignore-next-line */
+            function (ChildDefinition $definition, WorkflowInterface $attribute, \ReflectionClass $reflection) use ($workerAssignmentAttrExtractor): void {
+                $definition->addTag(
+                    'temporal.workflows',
+                    ['worker_name' => $workerAssignmentAttrExtractor($reflection)]
+                );
+            }
+        );
+
+        $container->registerAttributeForAutoconfiguration(
+            ActivityInterface::class,
+            /** @phpstan-ignore-next-line */
+            function (ChildDefinition $definition, ActivityInterface $attribute, \ReflectionClass $reflection) use ($workerAssignmentAttrExtractor): void {
+                $definition->addTag(
+                    'temporal.activities',
+                    [
+                        'worker_name' => $workerAssignmentAttrExtractor($reflection),
+                        'prefix' => $attribute->prefix,
+                    ]
+                );
+            }
+        );
+
+        $container->register(DataConverter::class, DataConverter::class)
+            ->setArguments($registerServiceArray($config['data_converters'] ?? null));
+
+        $container->setAlias('temporal.data_converter', DataConverter::class);
+        $container->setAlias(DataConverterInterface::class, 'temporal.data_converter');
+
+        $container
+            ->register(RebootKernelInterceptor::class)
+            ->addArgument(new Reference(KernelInterface::class))
+            ->addArgument(new Reference(LoggerInterface::class))
+            ->addTag('monolog.logger', ['channel' => BaldinofRoadRunnerExtension::MONOLOG_CHANNEL])
+        ;
+
+        $defaultInterceptors[] = RebootKernelInterceptor::class;
+
+        if (isset($bundles['DoctrineBundle'])) {
+            $container
+                ->register(DoctrineORMInterceptor::class)
+                ->addArgument(new Reference(ManagerRegistry::class))
+                ->addArgument(new Reference('service_container'))
+                ->addArgument(new Reference(EventDispatcherInterface::class))
+                ->addArgument(new Reference(LoggerInterface::class))
+                ->addTag('monolog.logger', ['channel' => BaldinofRoadRunnerExtension::MONOLOG_CHANNEL])
+            ;
+
+            $defaultInterceptors[] = DoctrineORMInterceptor::class;
+        }
+
+        $container->setParameter('temporal.default_interceptors', $defaultInterceptors);
+
+        foreach ($config['clients'] as $name => $options) {
+            $container->register("temporal.client.{$name}.connection", Connection::class)
+                ->setFactory([ConnectionFactory::class, 'createFromArray'])
+                ->setArguments([
+                    '$options' => [
+                        'address' => $options['address'],
+                        'crt' => $options['client_key'] ?? null,
+                        'client_key' => $options['client_key'] ?? null,
```

Comment:
Certificates should be configurable by itself

(currently it's client key that's used)

```suggestion
                        'crt' => $options['crt'] ?? null,
                        'client_key' => $options['client_key'] ?? null,
```